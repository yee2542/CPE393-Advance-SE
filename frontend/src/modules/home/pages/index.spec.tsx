import { shallow } from "enzyme";
import HomePage from "./home";

describe("Pages/index", () => {
  it("Should match snapshots", () => {
    const wrap = shallow(<HomePage />);
    expect(wrap).toMatchSnapshot();
  });
});
