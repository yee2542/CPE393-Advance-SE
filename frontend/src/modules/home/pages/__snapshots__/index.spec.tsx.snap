// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/index Should match snapshots 1`] = `ShallowWrapper {}`;
